app/controllers/user_filters_controller.rb

```diff
@@ -19,7 +19,8 @@ def create
 
     active_filter_options = params.require(:activeFilterOptions).collect do |filter|
       {
-        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, options: []),
+        filterOption: filter.require(:filterOption).permit(:name, :title, :description, :type, :hasTimestamp, :allowRange, :tooltip,
```

Comment:
What's the implications of this change? Do we need to update existing filters?

Reply:
All that was added here was the fields option, which is new with the multi type fields, and the tooltip.  I noticed in some testing that if you have a filter saved that should have a tooltip, it doesn't show up when you load the saved filter because it's not included in the save here.  We might want to refactor this eventually because we don't need to save most of these things but this fixes it for now